ledger/ledger-on-memory/src/main/scala/com/daml/ledger/on/memory/InMemoryLedgerReaderWriter.scala

```diff
@@ -59,6 +60,7 @@ final class InMemoryLedgerReaderWriter private (
   override def commit(correlationId: String, envelope: Bytes): Future[SubmissionResult] =
     committer.commit(correlationId, envelope, participantId)
 
+  @SuppressWarnings(Array("org.wartremover.warts.Any")) // so we can use `.view`
```

Comment:
This is a bit unfortunate, thanks for adding the comment.

Reply:
I am sad about this too.